lighthouse-core/gather/gatherers/css-usage.js

```diff
@@ -93,6 +103,24 @@ class CSSUsage extends FRGatherer {
       await this.stopCSSUsageTracking(context);
     }
 
+    const medias = await context.driver.executionContext.evaluate(getStylesheetMedias, {
+      args: [],
+      useIsolation: true,
+    });
+    this._stylesheets = this._stylesheets.filter(sheet => {
```

Comment:
This filter seems like something that should be applied in the audits. Does it really make sense to exclude the sheets here?